lib/rubanok/plane.rb

```diff
@@ -30,10 +30,10 @@ module Rubanok
   #
   # You can access the input data via `raw` method.
   class Plane
-    class << self
-      include DSL::Mapping
-      include DSL::Matching
+    extend DSL::Matching
```

Comment:
Let's avoid "changes" which do not change anything and don't relate to the current PR

Reply:
We really need this changes, because I need to add instance methods to `Rubanok::Rule`